### ISSUE TITLE
Accept social login

### ISSUE THREAD

**Issue by OmerSabic** [Project-Skill-Tree/Skill-Tree-API#93]
Once the base login is done, we need to add functionality for google and discord login options.

**Comment by TheEveEye** [comment]
I'm gonna split this into two issues: Discord Login #97 and Google Login #98

**Comment by OmerSabic** [comment]
> I'm gonna split this into two issues: Discord Login #97 and Google Login #98 

There is no need to split it. The Lucia implementation makes it extremely easy to add both and unnecessary to split it
